Cargo.toml

```diff
@@ -38,79 +39,86 @@ debug = true # Symbols are nice and they don't increase the size on Flash
 opt-level = "s"
 
 [workspace.dependencies]
-anyhow = "1.0.71"
+anyhow = "1.0.89"
 async-channel = "2"
 async-executor = "1"
 async-io = "2"
-async-lock = "3.3.0"
+async-lock = "3.4.0"
 async-std-openssl = "0.6.3"
 atomic-waker = "1.1.2"
-base64 = "0.21.0"
-bitfield = "0.14.0"
+base64 = "0.22.1"
+bitfield = "0.17.0"
 bytecodec = "0.4.15"
-bytes = "1.2.1"
+bytes = "1.7.2"
 cargo_metadata = "0.18.1"
-cbindgen = "0.26.0"
-chrono = "0.4.31"
-chrono-tz = "0.8.4"
-clap = "4.5.4"
-crc32fast = "1.3.2"
+cbindgen = "0.27.0"
+chrono = "0.4.38"
+chrono-tz = "0.10.0"
+clap = "4.5.20"
+crc32fast = "1.4.2"
 dns-message-parser = { version = "0.7", default-features = false}
-dialoguer = "0.10.4"
-either = "1.8.0"
-embedded-hal = { version = "0.2.7", features = ["unproven"] }
+dialoguer = "0.11.0"
+either = "1.13.0"
+# TODO(RSDK-8991): Upgrade to latest `embedded-hal`
+embedded-hal = { version = "=0.2", features = ["unproven"] }
 embedded-svc = "0.27"
-embuild = "0.31.3"
-env_logger = "0.10.1"
-esp-idf-svc = { version = "=0.48.1", default-features = false }
+embuild = "0.32.0"
+env_logger = "0.11.5"
+# TODO(RSDK-8994): Upgrade to latest `esp-idf-svc`
+esp-idf-svc = { version = "=0.48", default-features = false }
 esp-idf-part = "0.5.0"
 espflash = { git = "https://github.com/viamrobotics/espflash.git", branch = "monitor-output" }
-futures = "0.3.28"
-futures-lite = "1"
-futures-rustls = "0.24"
-futures-util = "0.3.30"
-http-body-util = "0.1.1"
+futures = "0.3.31"
+futures-lite = "2"
+# TODO(RSDK-8995): Upgrade to latest `rustls` infrastructure
+futures-rustls = "=0.24"
+futures-util = "0.3.31"
+http-body-util = "0.1.2"
 hyper = { version = "1.4", default-features = false, features = ["server", "client", "http2"] }
-ignore = "=0.4.20"
-lazy_static = "1.4.0"
-local-ip-address = "0.5.3"
-log = "0.4.20"
+ignore = "0.4"
+lazy_static = "1.5.0"
```

Comment:
should we add a TODO to use the std versions and remove this dependency?

Reply:
Done: https://viam.atlassian.net/browse/RSDK-9045